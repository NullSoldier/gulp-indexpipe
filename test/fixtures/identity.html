<!-- build:js scripts -->
<script src="data/lib.js"></script>
<script src="data/main.js"></script>
<!-- endbuild -->

<!-- build:css styles -->
<link rel="stylesheet" href="data/lib.css"/>
<link rel="stylesheet" href="data/main.css"/>
<!-- endbuild -->
