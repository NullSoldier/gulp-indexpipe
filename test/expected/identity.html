<script src="data/lib.js"></script>
<script src="data/main.js"></script>

<link rel="stylesheet" href="data/lib.css"/>
<link rel="stylesheet" href="data/main.css"/>
